presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -40,7 +40,7 @@
  * so some test cases are overridden from the base class and slightly modified to add an additional UUID column.
  */
 public class TestAccumuloDistributedQueries
-        extends AbstractTestDistributedQueries
+        extends AbstractConnectorReadWriteTest
```

Comment:
There seem to be much more than just read writes in `AbstractTestDistributedQueries` test, e.g `coercions, aggregation pushdown, sessions, create tables`. 

Should this be called `AbstractConnectorIntegrationTest`?

Seems like some tests are not needed here, e.g: `io.prestosql.testing.AbstractTestDistributedQueries#testSetSession`